index.html

```diff
@@ -139,6 +142,21 @@ <h2>Methods</h2>
         </dd>
       </dl>
     </section>
+    <section>
+      <h2>Modifications to existing procedures</h2>
+
+      <p>Append the following two paragraphs to the description of
+      <a data-cite="WEBRTC#dom-peerconnection-addicecandidate">addIceCandidate</a>:</p>
```

Comment:
We might move the algorithm to a neutral chapter in a separate editorial PR, so I'd remove mention of addIceCandidate here.
```suggestion
      <p>Append the following paragraph to the
      <a data-cite="WEBRTC#dfn-administratively-prohibited">administratively prohibited</a>
      algorithm:</p>
```

Reply:
Done.